recipe/meta.yaml

```diff
@@ -27,7 +27,7 @@ requirements:
     - wheel
     - setuptools
   run:
-    - ordered-set >=4.0.2,<4.2.0
+    - ordered-set >=4.1.0,<4.2.0
     - orjson
```

Comment:
`orjson` is optional since v6-3-0, but we delivered it in previous version v6-7-1, it's an optimization, so I suppose it can stay.